config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   resources :freelancers, only: [:index]
+  resources :cart, only: [:create]
```

Comment:
you should be able to add show here and remove line 5

Reply:
the user story says the path should be '/cart', so i think we need the custom route. If we add a show here, the route will be '/cart/:id'